src/SFA.DAS.Functions.Importer/Extensions/AddConfigurationExtension.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Extensions.Configuration;
+using SFA.DAS.Configuration.AzureTableStorage;
+using System.IO;
+
+namespace SFA.DAS.Functions.Importer.Extensions;
+
+public static class AddConfigurationExtension
+{
+    public static void AddConfiguration(this IConfigurationBuilder builder)
+    {
+        builder
+            .SetBasePath(Directory.GetCurrentDirectory())
+            .AddJsonFile("local.settings.json", optional: true);
+
+        var config = builder.Build();
+
+        builder.AddAzureTableStorage(options =>
+        {
+            options.ConfigurationKeys = config["ConfigNames"].Split(",");
```

Comment:
There is a compile time warning here. Please make sure there are no compile time warnings.